Sources/Network/Certificates+Extensions.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+import Foundation
+
+extension Data {
+    public func asPinnedCertificate() -> Result<SecCertificate, InvalidInputError> {
```

Comment:
Just minor naming feedback here - 

These functions are general cert functions, but the fact that a cert is a pinned cert really just meta-info about how we're using the cert...

Nothing about the fact that the cert is being used as a pinned cert matters here, right?  Can we drop the 'pinned' from these function names?

Reply:
makes sense, Ill change to asSecCertificate and pinnedKeys -> secKeys/asSecKeys